front_end/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
won't let me comment on `front_end/package-lock.json`, but can you remove that file and add it to the .gitignore?